modules/ROOT/pages/provisioning-ibmcloud.adoc

```diff
@@ -0,0 +1,106 @@
+
+Regarding the `ibmcloud` CLI, it is worth noting that it is supported to run the CLI https://cloud.ibm.com/docs/cli?topic=cli-using-idt-from-docker[via a container]. You'll need both the `cloud-object-storage` and `infrastructure-service` plugins installed. This can be done with:
+
+ * `ibmcloud plugin install cloud-object-storage`
+ * `ibmcloud plugin install infrastructure-service`
+
+After you've logged in using `ibmcloud login` you can set a target region:
+
+.Target a specific region
+[source, bash]
+----
+REGION='us-east' # run `ibmcloud regions` to view options
+ibmcloud target -r $REGION
+----
+
+=== Creating an Image
+
+The following sets of commands will show you how to download the most recent image for a stream, upload it to cloud storage, and then create the cloud image in IBM Cloud. It is worth noting that Fedora CoreOS comes in three streams, with different update schedules per stream. These steps show the `stable` stream as an example, but can be used for other streams too.
+
+
+.Fetch the latest image suitable for your target stream (or https://getfedora.org/coreos/download/[download and verify] it from the web).
+[source, bash]
+----
+STREAM='stable'
+coreos-installer download -s "${STREAM}" -p ibmcloud -f qcow2.xz --decompress
+----
+
+.Create a Service Account for uploading and an Authorization Policy to allow creating images from the uploaded objects.
+[source, bash]
+----
+BUCKET='my-unique-bucket'
+ibmcloud resource service-instance-create "${BUCKET}-service-instance" cloud-object-storage standard global
+
+SERVICE_INSTANCE_ID='25df0db0-89a4-4cb8-900f-ed8b44259f80' # from just created service account
+ibmcloud iam authorization-policy-create is --source-resource-type image cloud-object-storage Reader --target-service-instance-id $SERVICE_INSTANCE_ID 
+----
+
+.Upload the fetched image file to IBM Cloud Object Storage.
+[source, bash]
+----
+FCOS_VERSION='...'
+FILE=fedora-coreos-${FCOS_VERSION}-ibmcloud.x86_64.qcow2
+ibmcloud cos create-bucket --bucket $BUCKET --ibm-service-instance-id $SERVICE_INSTANCE_ID
+ibmcloud cos upload --bucket=$BUCKET --key=$FILE --file=$FILE
+----
+
+.Create the image from the storage object.
+[source, bash]
+----
+IMAGE=${FILE:0:-6}     # pull off .qcow2
+IMAGE=${IMAGE//[._]/-} # replace . and _ with -
+ibmcloud is image-create $IMAGE --file "cos://${REGION}/${BUCKET}/${FILE}" --os-name centos-7-amd64
+----
+
+NOTE: Specifying `--os-name centos-7-amd64` is required for now until IBM expands the list of OS types.
+
+You'll have to wait for the image creation process to and now show up as `pending`. Before you can use the image. Monitor with the following command:
```

Comment:
I think something went wrong with this sentence, possibly some copy-paste mixup?

Reply:
indeed :spaghetti:

will fix

Reply:
New wording in last push. Hopefully that is better.